HOSTING.md

```diff
@@ -109,8 +109,78 @@ The following describes thbe steps to deploy Vectara Answer using the Render API
            console.log(JSON.stringify(error))
        })
   ```
-  - Execute `node <command.js>`
+  - Execute `node command.js`
 
   This would create and deploy the vectara-answer service on Render. You can visit the [API docs](https://api-docs.render.com/reference/introduction) for more information.
+
+#### The following describes the steps to deploy Vectara Answer docker image using the Render API:
```

Comment:
Let's be consistent calling it "vectara-answer" (all lower case, with hyphen).
I would say "The following describes the steps required to deploy vectara-answer as a docker image to Render, by utilizing the Render API"

Reply:
Perhaps add before the steps the following:

"Note that this deployment mode pulls the latest docker container image from [docker-hub](https://hub.docker.com/repository/docker/vectara/vectara-answer/general)"